python/tests/integration/completions/e2e_text_completion.py

```diff
@@ -7,9 +7,10 @@
 logger = logging.getLogger()
 
 
-async def retry(func, retries=15, delay=1):
+async def retry(func, retries=15, delay=3):
     for i in range(retries):
         try:
+            time.sleep(delay)
```

Comment:
there is a sleep at line 22, after an error occurs. Why add another sleep before retrying? 

btw this will cause a sleep also before the 1st attempt

Reply:
Yeah, there are six calls to it in this file and I could only get it to not reach the rate limit by sleeping between each one.

Reply:
We should consider skipping the majority of the OpenAI tests like we do for the .NET SK

Reply:
@mkarle we can increase the delay after the error, which would be equivalent right? (without the need to wait before the first request)